### PR TITLE
Remember me

### PR DESCRIPTION
**Pull request by lucgilr**
Checkbox Remember_me OK

### CODE REVIEW

**Comment by JuliaJMP** [comment]
Ok :+1:

**Comment by Sermodi** [comment]
:+1:

**Comment by carloscharx** [comment]
:100:

File: app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 1]
```diff
@@ -58,6 +61,9 @@ public void onResponse(LoginModel response) {
             mActivity.getApplication();
             Intent intent = new Intent(mContext, MainActivity.class);
             intent.putExtra("token", response.getToken());
+            if (mCheckBox.isChecked()){
+                intent.putExtra(getString(R.string.remember_checked_intent), true);
```
Comment (jrodgut-sg) [inline app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: it's not needed to place it in the android strings, a java variable is better as it avoid problems with internationalization

Comment (jrodgut-sg) [inline app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: it is not needed to place this in the Android Strings, a java variable would be enough

File: app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 2]
```diff
@@ -20,15 +25,21 @@ protected void onResume() {
...
 
         Bundle extras = getIntent().getExtras();
-        String token = null;
-        if (extras != null) {
-            token = extras.getString("token", null); /** TODO: Read token when it will be stored **/
+        String token = RememberUtils.getToken(this);
+        boolean isRememberChecked = false;
+        if (extras != null && token == null) {
+            token = extras.getString("token", null);
+            isRememberChecked = extras.getBoolean(getString(R.string.remember_checked_intent), false);
```
Comment (jrodgut-sg) [inline app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: it is not needed to place this in the Android Strings, a java variable would be enough

**Comment by garciparedes** [comment]
:+1:

**Pull request merged**